lnbits/__init__.py

```diff
@@ -15,7 +15,7 @@
 disabled_extensions = getenv("LNBITS_DISABLED_EXTENSIONS", "").split(",")
 
 app = Flask(__name__)
-app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1, x_host=1)
+app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1, x_host=1) # type: ignore
```

Comment:
Replacing methods like this seems hacky and unsupported by mypy but apparently works